.travis.yml

```diff
@@ -68,11 +71,11 @@ jobs:
         - export CXX=g++-4.8 CC=gcc-4.8
       before_cache:
         - ls -t regression/install | tail -n+2 | sed s@^@regression/install/@ | xargs rm -rf
-    - stage: prepare cache-verilator
+    - stage: set up verilator
       script:
-        - travis_wait 120 make verilator -C regression SUITE=none
-      before_install:
-        - export CXX=g++-4.8 CC=gcc-4.8
+        # This installs into /usr/local/bin/verilator
+        - curl -L -O verilator.deb https://github.com/sifive/verilator/releases/download/4.028-0sifive1/verilator_4.028-0sifive1_amd64.deb
```

Comment:
presumably the "make verilator" target was picking the version somehow? Should we make those two versions line up (the one this curl uses and the one you'd get if you decided to run "make verilator"?

Reply:
That sounds reasonable, but I can't think of a good way of doing this besides just manually making sure that these version numbers are updated in both places. Do you have any suggestions?

Reply:
could we put the version number in a shared location (like "verilator-version.txt") and use that whenever we need it? Similar to the rocket tools hash file at the top level?

Reply:
That will rely on me/SiFive not changing the package naming convention for the SiFive hosted .deb package, which I guess will at least be true until we start building packages for other .deb-based distros, like Debian or other versions of Ubuntu. At least if it changes, it'll error out.

I'll make this change (if I can get the stupid build to work at all).